content/applications/hr/appraisals/goals.rst

```diff
@@ -0,0 +1,73 @@
+=====
+Goals
+=====
+
+The ability to set goals for employees to work toward is an important feature of Odoo *Appraisals*.
+Goals are typically set during an appraisal, so the employee knows what to work toward before their
+next appraisal.
+
+To view all goals, navigate to :menuselection:`Appraisals application --> Goals`. This presents all
+the goals for every employee, in a default Kanban view.
+
+Each goal card contains the following information:
+
+- :guilabel:`Goal`: the name of the goal.
+- :guilabel:`Employee`: the employee the goal is assigned to.
+- :guilabel:`Activities`: displays the corresponding :doc:`activity icon
```

Comment:
i was confused by this, as it was not the word "Activities" shown in the card, but the activities icon.

Maybe we can change this to the clock icon instead?
```suggestion
- :icon:`fa-clock-o` (:guilabel:`clock`) icon: displays the corresponding :doc:`activity icon
```

Reply:
Sure- it's hard to define since it can appear as several icon- whatever is scheduled is what appears (a group of people, an envelope, etc). But I'll use the clock here!